src/azure-cli/azure/cli/command_modules/acs/_loadbalancer.py

```diff
@@ -59,18 +59,22 @@ def configure_load_balancer_profile(managed_outbound_ip_count, outbound_ips, out
     outbound_ip_resources = _get_load_balancer_outbound_ips(outbound_ips)
     outbound_ip_prefix_resources = _get_load_balancer_outbound_ip_prefixes(outbound_ip_prefixes)
 
-    if managed_outbound_ip_count:
-        profile.managed_outbound_ips = ManagedClusterLoadBalancerProfileManagedOutboundIPs(
-            count=managed_outbound_ip_count
-        )
-    if outbound_ip_resources:
-        profile.outbound_ips = ManagedClusterLoadBalancerProfileOutboundIPs(
-            public_ips=outbound_ip_resources
-        )
-    if outbound_ip_prefix_resources:
-        profile.outbound_ip_prefixes = ManagedClusterLoadBalancerProfileOutboundIPPrefixes(
-            public_ip_prefixes=outbound_ip_prefix_resources
-        )
+    if managed_outbound_ip_count or outbound_ip_resources or outbound_ip_prefix_resources:
+        profile.managed_outbound_ips = None
```

Comment:
just to confirm behavior, when any of `managed_outbound_ip_count`, `outbound_ip_resources`, `outbound_ip_prefix_resources` from customer inputs, all others will be set to None?

Reply:
@JunSun17 

you can use multiple assignment in python and set them all to none in one go

x, y, z = None, None, None

FYI don't do x = y = z = None or I think it will assign the values to the same block of memory 

https://note.nkmk.me/en/python-multi-variables-values/